web/js/link/link.model.js

```diff
@@ -1,7 +1,11 @@
-var wv = wv || {};
-wv.link = wv.link || {};
+import $ from 'jquery';
+import each from 'lodash/each';
+import map from 'lodash/map';
+import isArray from 'lodash/isArray';
+import isObject from 'lodash/isObject';
+import util from '../util/util';
 
-wv.link.model = wv.link.model || function (config) {
+export function linkModel(config) {
```

Comment:
What do you think about keeping the name `model`, importing this into the `link` parent, and attaching it to the object that exports?